<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWindow</class>
 <widget class="QWidget" name="AboutWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>О программе</string>
  </property>
  <widget class="QLabel" name="WebSiteLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Веб-сайт</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProgramVersionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Версия</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmailLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Поддержка</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProgramVersion">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2024.04.04</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>20</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/Frame.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="UrlPrivacyPolicy">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>184</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Политика конфиденциальности</string>
   </property>
  </widget>
  <widget class="QLabel" name="Copyright">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>224</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Copyright © 2024
Simonovskiy &amp; Lastivka
All rights reserved</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="WebSite">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>controlmictray.pp.ua</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Email">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>190</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>info@controlmictray.pp.ua </string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoFrame">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>26</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/Microphone.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
